packages/vue/src/components/sign-in.vue

```diff
@@ -134,27 +129,24 @@ const onForgotPasswordClicked = (): void => {
                 : signInButtonText
             }}
           </amplify-button>
-          <amplify-button
-            class="amplify-field-group__control"
-            data-fullwidth="true"
-            data-size="small"
-            data-variation="link"
-            style="font-weight: normal"
-            type="button"
-            @click.prevent="onForgotPasswordClicked"
-          >
-            {{ forgotYourPasswordLink }}
-          </amplify-button>
         </base-wrapper>
-        <base-footer>
-          <slot
-            name="footer"
-            :onSignInSubmit="onSignInSubmit"
-            :onForgotPasswordClicked="onForgotPasswordClicked"
-          >
-          </slot>
-        </base-footer>
       </base-form>
     </base-wrapper>
+
+    <base-footer>
+      <slot name="footer">
```

Comment:
Footer is outside the form 👇  with the default "Reset Password" link

Reply:
👍